lib/gettext-setup/pot.rb

```diff
@@ -137,5 +137,19 @@ def self.update_pot(locales_path = GettextSetup.locales_path, path = nil)
         end
       end
     end
+
+    def self.merge_pot(locales_path = GettextSetup.locales_path)
+      mergepot = GettextSetup.config['project_name'] + '.pot'
```

Comment:
Is there a reason to use this rather than something generic like `merged.pot`? My understanding of this functionality is that it's a bit of a meta-task and could cross projects, no?

Whichever path is used, I'd suggest extracting it to a variable inside the Pot class, e.g. `GettextSetup::Pot.merged_path` for use in e.g. tests.